contracts/MANAContinuousSale.sol

```diff
@@ -0,0 +1,53 @@
+pragma solidity ^0.4.11;
+
+import "zeppelin-solidity/contracts/ownership/Ownable.sol";
+import "./ContinuousSale.sol";
+
+contract MANAContinuousSale is ContinuousSale, Ownable {
+
+    uint256 public constant INFLATION = 8;
+
+    bool public started = false;
+
+    event RateChange(uint256 amount);
+
+    event WalletChange(address wallet);
```

Comment:
Yes, there is quite a bit of duplication here, showcasing the need for a more barebones Crowdsale base contract.